cmd/disk-cache-config.go

```diff
@@ -52,12 +55,42 @@ func (cfg *CacheConfig) UnmarshalJSON(data []byte) (err error) {
 
 // Parses given cacheDrivesEnv and returns a list of cache drives.
 func parseCacheDrives(drives []string) ([]string, error) {
+	if len(drives) == 0 {
+		return drives, nil
+	}
+	var endpoints []string
 	for _, d := range drives {
+		if ellipses.HasEllipses(d) {
+			s, err := parseCacheDrivePaths(d)
+			if err != nil {
+				return nil, err
+			}
+			endpoints = append(endpoints, s...)
+		} else {
+			endpoints = append(endpoints, d)
+		}
+	}
+
+	for _, d := range endpoints {
```

Comment:
shouldn't we avoid this loop by checking `filepath.IsAbs(d)` before appending to endpoints slice itself?

Reply:
parseCacheDrivePaths returns a slice of drives, it looks cleaner to avoid nested for loop

Reply:
I guess rather than failing we could perhaps get an Abs path? i.e use `filepath.Abs` - why did we decide to enforce this?

Reply:
>  it looks cleaner to avoid nested for loop

okay, I did not consider the nested loop

Reply:
As cache drives are specifed through env variables, it would be hard to guess user's intent if relative path is allowed as per discussion with @harshavardhana.